docs-source/content/samples/simple/azure-kubernetes-service/includes/create-aks-cluster-body-01.txt

```diff
@@ -0,0 +1,82 @@
+##### Create a Service Principal for AKS
+
+An AKS cluster requires either an [Azure Active Directory (AD) service principal](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals) or a [managed identity](https://docs.microsoft.com/azure/aks/use-managed-identity) to interact with Azure resources.
+
+We will use a service principal to create an AKS cluster. Follow the commands below to create a new service principal.
+
+Please run `az login` first. Do set the subscription you want to work with. You can get a list of your subscriptions by running `az account list`.
+
+```bash
+# Login
+$ az login
+
+# Set your working subscription
+$ export SUBSCRIPTION_ID=<your-subscription-id>
+$ az account set -s $SUBSCRIPTION_ID
+```
+
+Create the new service principal with the following commands:
+
+```bash
+# Create Service Principal
+$ export SP_NAME=myAKSClusterServicePrincipal
+$ az ad sp create-for-rbac --skip-assignment --name $SP_NAME
+
+# Copy the output to a file, we will use it later.
+```
+
+If you see an error similar to the following:
+
+```bash
+Found an existing application instance of "5pn2s201-nq4q-43n1-z942-p9r9571qr3rp". We will patch it
+Insufficient privileges to complete the operation.
+```
+
+The problem may be a pre-existing service principal with the same name.  Either delete the other Service Principal or pick a different name.
```

Comment:
service principal or Service Principal (I'm not 100% sure which it should be (I think capitalized), but whichever is correct, please use it consistently.)